### PR TITLE
Update Add-SPOSiteCollectionAppCatalog.md

### PR DESCRIPTION
**Pull request by lucabandMSFT**
provided updates on the permissions required to successfully run the cmdlet

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ Add-SPOSiteCollectionAppCatalog -Site <SpoSitePipeBind> [-WhatIf] [-Confirm] [<C
 
 ## DESCRIPTION
 
-Use this cmdlet to add a site collection scoped app catalog to the specified site.
+Use this cmdlet to add a site collection scoped app catalog to the specified site.To run this cmdle user must have the following roles:
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:27]: Missing a 't' in 'cmdlet'.

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ Add-SPOSiteCollectionAppCatalog -Site <SpoSitePipeBind> [-WhatIf] [-Confirm] [<C
...
+* SharePoint tenant administrator
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:28]: This should be 'SharePoint Admin role or Global Administrator role'.

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md [hunk 3]
```diff
@@ -24,7 +24,10 @@ Add-SPOSiteCollectionAppCatalog -Site <SpoSitePipeBind> [-WhatIf] [-Confirm] [<C
...
+* site collection administrator of the tenant app catalog
+* site collection administrator of the site collection of the site specified in the `-Site` parameter
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:30]: Generally in docs I see "Site Collection Administrator" capitalized.

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by get-itips — CHANGES_REQUESTED** [review]
agree with Trevor

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md (hunk 1 above)
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:27]:
```suggestion
Use this cmdlet to add a site collection scoped app catalog to the specified site. To run this cmdlet user must have the following roles:
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md (hunk 2 above)
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:28]:
```suggestion
* SharePoint Admin role or Global Administrator role
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md (hunk 3 above)
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:30]:
```suggestion
* Site Collection Administrator of the tenant app catalog
* Site Collection Administrator of the site collection of the site specified in the `-Site` parameter
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Agree with @tseward.

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md (hunk 1 above)
Comment (lucabandMSFT) [reply sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md]: thanks!

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md (hunk 2 above)
Comment (lucabandMSFT) [reply sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md]: agreed, Thanks

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md (hunk 3 above)
Comment (lucabandMSFT) [reply sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteCollectionAppCatalog.md:30]: thanks for double checking it

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Thanks @lucabandMSFT!

**Review by get-itips — APPROVED** [review]
@Kateyanne ready to be merged

**Pull request merged**